app/assets/stylesheets/utilities/layout.scss

```diff
@@ -1,3 +1,11 @@
 .float-right {
   float: right;
 }
+
+.d-flex {
+  display: flex;
+}
+
+.justify-content-between {
+  justify-content: space-between;
+}
```

Comment:
I haven't added this to application_ie8.scss as flexbox is not supported. In ie8 it will display on a new line as recommended in the Gov.UK Design System: https://design-system.service.gov.uk/components/header/with-service-name-and-navigation/index.html